test/bigtest/network/config.js

```diff
@@ -20,23 +30,110 @@ export default function configure() {
 
   this.get('/users', {});
 
-  this.post('/bl-users/login', () => {
-    return new Response(201, {
-      'X-Okapi-Token': `myOkapiToken:${Date.now()}`
-    }, {
-      user: {
-        id: 'test',
-        username: 'testuser',
-        personal: {
-          lastName: 'User',
-          firstName: 'Test',
-          email: 'user@folio.org',
-        }
-      },
-      permissions: {
-        permissions: []
-      }
-    });
+  this.post('/bl-users/login', (schema, request) => {
+    const formData = JSON.parse(request.requestBody);
+
+    switch (formData.username) {
```

Comment:
Mirage route definitions can be overridden in individual tests. I'd recommend redefining the `bl-users/login` endpoint in each individual test that needs a different response, instead of the `default` scenario relying on different inputs.

Here's an example: https://github.com/folio-org/ui-eholdings/blob/15dbfb7a1a73b720834fc12cc3bf8105bd30e406/test/bigtest/tests/resource-show-test.js#L190-L194